### PR TITLE
External properties restrictions

### PR DESCRIPTION
**Pull request by zvr**
Adds restrictions to external properties

The way to specify restrictions to external properties is:

- refer to the external property as a complete path of /{profile}/{class}/{property}
- add the shape defintion, which overwrites the original one.

Realistically, the restrictions will mostly be optional properties becoming mandatory,
i.e., minCount changing from 0 to 1.

### CODE REVIEW

**Review by goneall — APPROVED** [review]
LGTM

File: model/Core/Classes/SpdxDocument.md [hunk 1]
```diff
@@ -24,3 +24,8 @@ Commonly used when representing a unit of transfer of SPDX Elements.
   - minCount: 1
   - maxCount: 1
 
+## External properties restrictions
+
+- /Core/Element/name
```
Comment (maxhbr) [inline model/Core/Classes/SpdxDocument.md:29]: you use `/` as seperator and in other places we use `:`:

https://github.com/spdx/spdx-3-model/blob/01360996d37c1d913f3d15e2d5911ace182ed200/model/Software/Classes/File.md?plain=1#L16

Can we make this consistent?

**Review by maxhbr — CHANGES_REQUESTED** [review]

File: model/Core/Classes/SpdxDocument.md (hunk 1 above)
Comment (maxhbr) [reply model/Core/Classes/SpdxDocument.md:29]: It would probably be best, to align with the way rdf is referencing the properties.

**Review by goneall — APPROVED** [review]
LGTM

**Pull request merged**